3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -15,29 +15,44 @@
    url with `.shx`. There is no server at the modified url, therefore this 
    should result in a network (DNS) error.
 ------------------------------------------------------------------------------*/
-function requestData(url) {
+async function requestData(url) {
   // TODO return a promise using `fetch()`
+  const response = await fetch(url);
+  const json = await response.json();
+  return json;
 }
 
 function renderImage(data) {
   // TODO render the image to the DOM
-  console.log(data);
+  const image = data.img
+  const img = document.createElement("img")
+  document.body.appendChild(img).src = image
 }
 
 function renderError(error) {
   // TODO render the error to the DOM
-  console.log(error);
+  if (error) {
```

Comment:
You don't need this `if` statement here. We already know that there is an error because that's why we are calling this function in line 54 in the first place.